assets/templates/partials/calendar/items/list.tmpl

```diff
@@ -9,7 +9,7 @@
       </a>
       <div class="ons-u-mt-s">
         <span class="ons-u-fs-r--b">Release date:</span>
-        <span>{{ .Description.ReleaseDate }}</span>
+        <span>{{ dateTimeFormat .Description.ReleaseDate }}</span>
         <span>|</span>
```

Comment:
Clarifying question: Are there now two dates being rendered here? Both the `dateTimeFormat` and the `.Description.ReleaseDate`?